lib/adyen/services/payments.rb

```diff
@@ -3,7 +3,7 @@
 module Adyen
   class Payments < Service
     attr_accessor :version
-    DEFAULT_VERSION = 40
+    DEFAULT_VERSION = 46
```

Comment:
Can we change PAL to v49 like in other libraries or do we need additional steps?
https://github.com/Adyen/adyen-java-api-library/blob/d70c84a478be3650b2afbc897edfdbe449aba005/src/main/java/com/adyen/Client.java#L38